recipe/upload_or_check_non_existence.py

```diff
@@ -16,19 +16,24 @@
 from conda_build.config import Config
 
 
+def bldpkg_path_wrapper(meta):
+    try:
+        config = Config()
+        fname = bldpkg_path(meta, config)
+    except TypeError:
+        # conda-build < 2.0.0 takes a single config argument
```

Comment:
If conda-build 2 behaves the same, doesn't this PR become useless along with PR #34 ?